operator/.codegen.yaml

```
@@ -1,2 +1,6 @@
+schemapatch:
+  requiredFeatureSets:
+    - ""
+    - "TechPreviewNoUpgrade"
```

Can you add Default and CustomNoUpgrade too please
```suggestion
    - ""
    - "Default"
    - "TechPreviewNoUpgrade"
    - "CustomNoUpgrade"
```